compiler/rustc_target/src/spec/targets/armv7a_vex_v5.rs

```diff
@@ -15,11 +15,12 @@ pub(crate) fn target() -> Target {
         data_layout: "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64".into(),
         arch: "arm".into(),
         options: TargetOptions {
-            abi: "eabi".into(),
+            cpu: "cortex-a9".into(),
+            abi: "eabihf".into(),
             linker_flavor: LinkerFlavor::Gnu(Cc::No, Lld::Yes),
             linker: Some("rust-lld".into()),
             link_script: Some(LINK_SCRIPT.into()),
-            features: "+v7,+thumb2,+soft-float,-neon,+strict-align".into(),
+            features: "+v7,+thumb2,+vfp3,+neon".into(),
```

Comment:
Note that enabling "+neon" will cause what we consider to be unsound float results due to https://github.com/rust-lang/rust/issues/129880

I do not necessarily think we should block this as a tier 3 target because of this, as this kind of problem hasn't even been resolved for all tier 2 targets, but be aware we may expect this to change later.

Reply:
I'm not sure if there's anything actionable I can do here personally other than remove the `neon` feature from this target (which is 100% something that I'd be willing to do in the name of soundness), but based on https://github.com/rust-lang/portable-simd/issues/439 it seems like this is more an underlying bug with LLVM codegen making assumptions about IEEE compliance that it shouldn't. The V5 is capable of emitting NEON instructions, I'm just not sure the typical policy here regarding codegen bugs like this in targets.